providers-sdk/v1/plugin/start.go

```diff
@@ -46,6 +47,15 @@ type Connector struct {
 func Start(args []string, impl ProviderPlugin) {
 	logger.CliCompactLogger(logger.LogOutputWriter)
 	zerolog.SetGlobalLevel(zerolog.WarnLevel)
+	if len(args) > 1 {
```

Comment:
we would need a proper flag for that, otherwise it will be difficult to add more flags without making a breaking change when we want to change the order